src/mmw/js/src/modeling/views.js

```diff
@@ -706,34 +704,43 @@ var BarChartView = Marionette.ItemView.extend({
         return 'bar-chart-' + this.model.get('name');
     },
     className: 'chart-container',
+
+    initialize: function() {
+        this.listenTo(this.model, 'change', this.addChart);
+    },
+
     onAttach: function() {
         this.addChart();
     },
+
     addChart: function() {
-        //TODO use real model results
-        var selector = '#' + this.id() + ' .bar-chart',
-            fakeStackedData = [
-                {
-                    type: 'Original',
-                    infiltration: 0.8,
-                    runoff: 0.1,
-                    evap: 0.1
-                },
-                {
-                    type: 'Modified',
-                    infiltration: 0.4,
-                    runoff: 0.2,
-                    evap: 0.4
-                }
-            ],
-            options = {
-                barColors: ['#329b9c', '#4aeab3', '#4ebaea'],
-                depAxisLabel: 'Level',
-                depDisplayNames: ['Infiltration', 'Runoff', 'Evaporation']
-            },
-            indVar = 'type',
-            depVars = ['infiltration', 'runoff', 'evap'];
-        chart.makeBarChart(selector, fakeStackedData, indVar, depVars, options);
+        var selector = '#' + this.id() + ' .bar-chart';
+        $(selector).empty();
+        var result = this.model.get('result');
+        if (result) {
+            var indVar = 'type',
+                depVars = ['inf', 'runoff', 'et'],
+                data = [
+                    {
+                        type: 'Original',
+                        inf: result['unmodified']['inf'],
+                        runoff: result['unmodified']['runoff'],
+                        et: result['unmodified']['et']
+                    },
+                    {
+                        type: 'Modified',
+                        inf: result['modified']['inf'],
+                        runoff: result['modified']['runoff'],
+                        et: result['modified']['et']
+                    }
+                ],
+                options = {
+                    barColors: ['#329b9c', '#4aeab3', '#4ebaea'],
```

Comment:
Not sure if we'll be using these elsewhere but we might want to store these as constants and add some comment about what what color each one actually is.